### ISSUE TITLE
Issue with table wrapper in main.js

### ISSUE THREAD

**Issue by GraydonGit** [TryGhost/Source#41]
Hi there,

I ran into a problem when I included more than four tables in a single blog post; they all appeared clustered together. It was quite odd, but I believe I've discovered a workaround.

At the end of the file changed: 

main.js:
```
/* Responsive HTML table */
(function () {
    const tables = document.querySelectorAll('.gh-content > table:not(.gist table)');
    const wrapper = document.createElement('div');
    wrapper.className = 'gh-table';

    tables.forEach(function (table) {
        table.parentNode.insertBefore(wrapper, table);
        wrapper.appendChild(table);
    });
})();
```

TO:

```
/* Responsive HTML table */
(function () {
    const tables = document.querySelectorAll('.gh-content > table:not(.gist table)');

    tables.forEach(function (table) {
        const wrapper = document.createElement('div');
        wrapper.className = 'gh-table';
        table.parentNode.insertBefore(wrapper, table);
        wrapper.appendChild(table);
    });
})();
```

**Comment by gituliar** [comment]
+1 Facing the same issue.

**Comment by minimaluminium** [comment]
Thanks @GraydonGit for the proposed fix. This is now included in Source 1.2.2.
